internal/worker/runstatic.go

```diff
@@ -77,7 +80,14 @@ func RunStaticAnalyses(pkg *pkgecosystem.Pkg, sbOpts []sandbox.Option, tasks ...
 
 	status := analysis.StatusForRunResult(runResult)
 
-	// TODO log status
+	totalTime := time.Since(startTime)
+	log.Info("Static analysis finished",
+		"ecosystem", pkg.EcosystemName(),
+		"name", pkg.Name(),
+		"version", pkg.Version(),
+		"status", string(status),
```

Comment:
The name/key `"status"` is ambiguous and probably should include some more context